roman-numerals-solution/RomanNumeralsConvert.java

```diff
@@ -0,0 +1,167 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class RomanNumeralsConvert {
+	
+	public final static String onesArray[] = { "I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX" };
+	public final static String tensArray[] = { "X", "XX", "XXX", "XL", "L", "LX", "LXX", "LXXX", "XC" };
+	public final static String hundredsArray[] = { "C", "CC", "CCC", "CD", "D", "DC", "DCC", "DCCC", "CM" };
+
+	public final static String TEST_MOD = "-test";
+	public final static String ONE = "I";
+	public final static String TWELVE = "XII";
+	public final static String ONE_HUNDRED_AND_THREE = "CIII";
+
+	private static String[] numArray;
+
+	public static void main(String[] args) {
+
+		if (args.length > 0) {
+			if (args.length > 1 || !TEST_MOD.equals(args[0])) {
+				System.out.println("Only one flag supports: - test");
+			} else {
+				runTest();
+			}
+		} else {
+			readInput();
```

Comment:
I like this style, but the methods should return something and the next method should use it as input.